lib/rust/ensogl/core/src/display/symbol/gpu/geometry/compound/mask_composer.rs

```diff
@@ -43,9 +43,11 @@ impl MaskComposer {
             vec4 sample_color = texture(input_{color},input_uv);
             vec4 sample_id    = texture(input_{id},input_uv);
             output_id         = (sample_mask.a > 0.5) ? sample_id : vec4(0.0,0.0,0.0,0.0);
+            // output_color      = (sample_mask.a > 0.5) ? sample_color : vec4(0.0,0.0,0.0,0.0);
             output_color      = sample_color;
             // We multiply all components by alpha because we store them in the premultiplied form.
             output_color *= sample_mask.a;
+            // output_color = sample_color;
             "
```

Comment:
During our last call I mentioned that I made some dirty changes to the codebase just to make it work, but these dirty changes need to be cleaned up. Here we have left comments that should be removed.